gemhardware/include/gem/hw/glib/GLIBManager.h

```diff
@@ -52,15 +52,13 @@ namespace gem {
           virtual void stopAction()       throw (gem::hw::glib::exception::Exception) override;
           virtual void haltAction()       throw (gem::hw::glib::exception::Exception) override;
           virtual void resetAction()      throw (gem::hw::glib::exception::Exception) override;
-          //virtual void noAction()         throw (gem::hw::glib::exception::Exception) override;
+          // virtual void noAction()         throw (gem::hw::glib::exception::Exception) override;
 
-          virtual void failAction(toolbox::Event::Reference e)
+          void failAction(toolbox::Event::Reference e)
```

Comment:
Is `virtual` removed because this function `override`s? If so add `override`.

Reply:
Not inherited (currently commented out in base class)